pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -461,6 +461,7 @@ func (cloud *CloudProvider) insertRegionalDisk(
 		}
 		return fmt.Errorf("unknown Insert disk operation error: %v", err)
 	}
+	klog.V(5).Infof("InsertDisk operation %s for disk %s", opName, diskToCreate.Name)
```

Comment:
sorry if I was not clear earlier. the error I meant was in line 421. So basically before we start waiting for the op in line 445. i.e put this log in line 444. That way we know what op was initiated, and check in cases of scenarios like op taking long time

Reply:
Ah ok got it.